changelog/unreleased/features/2162--environment-variables-as-config.md

```diff
@@ -0,0 +1,7 @@
+VAST has now complete support for passing environment variables as alternate
+path to configuration files. Environment variables have *lower* precedence than
+CLI arguments and *higher* precedence than config files. Variable names of the
+form `VAST_FOO__BAR_BAZ` map to `vast.foo.bar-baz`, i.e., `__` is a record
+separator and `_` translates to `-`. This does not apply to the prefix `VAST_`,
+which is considered the application identifier. Only variables with non-empty
+values are considered.
```

Comment:
Please create a corresponding docs PR before merging, and document that the list syntax differs from the list syntax on the command line.

Reply:
Turns out things work for lists. For example, `VAST_PLUGINS="sigma,pcap"` is equivalent to `--plugins=foo,bar`.